src/PayClient.ts

```diff
@@ -78,21 +81,52 @@ export class PayClient {
     return {
       method,
       url: `${this.apiUrl}${resource}`,
-      qs:  _.extend({appId, meta: true}, params),
+      qs: _.extend({ appId, meta: true }, params),
       body: payload ? JSON.stringify(payload) : undefined,
       timeout: this.config.timeout,
       headers: this.getHeaders(method, resource, payload),
       resolveWithFullResponse: true,
     };
   }
 
+  private getRequest (error: any, response: object | undefined, cb: ((error: any, object: object) => void) | null | undefined): any {
+    const body = _.get(response, 'body');
+    const headers = _.get(response, 'headers');
+    const status = _.get(response, 'statusCode');
+
+    if (cb) {
+      return cb(
+        error,
+        {
+          status,
+          object: (body && headers && headers['content-type'] &&
+            _.includes(headers['content-type'], 'application/json')) ?
+            JSON.parse(body)
+            : _.get(response, 'body'),
+        }
+      )
+    } else {
```

Comment:
You can remove this `else` and just have the `return` statement. Many style guides prefer this method.

Reply:
Updated the code.